sites/public/src/components/listings/Pagination.tsx

```diff
@@ -67,31 +72,48 @@ export function Pagination(props: PaginationProps) {
   // className={styles["pagination"]}
   // size={AppearanceSizeType.small}
 
+  if (props.lastPage === 1) return <></>
+
+  let buttonColumns = []
+
+  if (canNavBackward) {
```

Comment:
Only adds navigation buttons if there is a page to navigate to in that direction, as opposed to disabling it.

<img width="604" alt="Screenshot 2024-11-12 at 8 17 47 PM" src="https://github.com/user-attachments/assets/f6417f54-f638-44d2-865d-f621eec958c9">
<img width="601" alt="Screenshot 2024-11-12 at 8 18 58 PM" src="https://github.com/user-attachments/assets/134df6c5-4cd0-446e-bae0-dc2e9402a23d">